lightning/src/ln/wire.rs

```diff
@@ -16,6 +16,13 @@
 use ln::msgs;
 use util::ser::{Readable, Writeable, Writer};
 
+/// Maximum Lightning message data length according to
+/// [BOLT-8](https://github.com/lightningnetwork/lightning-rfc/blob/v1.0/08-transport.md#lightning-message-specification):
```

Comment:
I mean, its more because the length is encoded with two bytes than because the spec says so :p.

Reply:
or the spec uses two byte length encoding to enforce the message size limitation :)

Reply:
Done